src/compiler/interface2.jl

```diff
@@ -35,13 +35,17 @@ end
 end
 
 @generated function (j::Pullback{T})(Δ) where T
-  ignore_sig(T) && return :nothing
+  ignore_sig(T) && return :(DoesNotExist())
   g = try _lookup_grad(T)
   catch e
     rethrow(CompileError(T,e))
   end
   if g == nothing
-    Δ == Nothing && return :nothing
+    Δ <: AbstractZero && return :(DoesNotExist())
```

Comment:
```suggestion
    Δ <: AbstractZero && return :(Δ)
```

So that if it is `Zero()` or `DoesNotExist()` it stays that way?